### PR TITLE
vtk: fix generated cmake files to not reference C:/building

### PR DESCRIPTION
**Pull request by filnet**
see https://github.com/Alexpux/MINGW-packages/issues/1670

### CODE REVIEW

File: mingw-w64-gst-plugins-bad/PKGBUILD [hunk 1]
```diff
@@ -48,7 +48,8 @@ depends=("${MINGW_PACKAGE_PREFIX}-celt"
          "${MINGW_PACKAGE_PREFIX}-orc"
          "${MINGW_PACKAGE_PREFIX}-schroedinger"
          "${MINGW_PACKAGE_PREFIX}-soundtouch"
-         "${MINGW_PACKAGE_PREFIX}-openh264")
+         "${MINGW_PACKAGE_PREFIX}-openh264"
+		 )
```
Comment (Alexpux) [inline mingw-w64-gst-plugins-bad/PKGBUILD]: Always use SPACES instead TABS

Comment (filnet) [inline mingw-w64-gst-plugins-bad/PKGBUILD]: that commit is spurious and was removed

**Comment by Alexpux** [comment]
Well I don't remember why this is commented in the past but don't think that this solve problem

**Comment by filnet** [comment]
opencv was failing because of the hard wired C:/building in vtk cmake files.
with this PR there are no more hard wired paths in vtk cmake files and opencv compiles fines.

**Pull request merged**